docs/CHANGELOG.asciidoc

```diff
@@ -8,16 +8,3934 @@
+* Adds a new memory usage by job and by model view {kibana-pull}149419[#149419]
+* Allow Anomaly Detection geo jobs to be created from maps dashboard {kibana-pull}147797[#147797]
+* Adds geo fields support for Unified field list, add statistics flyover to Anomaly detection job creation wizards {kibana-pull}147322[#147322]
+* Anomaly Detection wizards: adds geo job wizard {kibana-pull}147043[#147043]
+
+Management::
+* Adds field statistics popovers for Data Frame Analytics & Transform creation wizards {kibana-pull}149879[#149879]
+* Transforms: Shows health status of transform in UI {kibana-pull}150359[#150359]
+
+Monitoring::
+* Adds duration configuration to Stack Monitoring Cluster Health rule {kibana-pull}147565[#147565]
+
+Observability::
+* Adds alert summary widget to overview page {kibana-pull}149581[#149581]
+* Adds AlertSummaryWidget full-size on the Alerts page {kibana-pull}148539[#148539]
+* Additional context for log threshold rule {kibana-pull}148503[#148503]
+* Adds charts to Alert Summary Widget {kibana-pull}148143[#148143]
+* Adds rule details locator and make AlertSummaryWidget clickable {kibana-pull}147103[#147103]
+* Adds groupByKeys context to recovered alerts for Log Threshold Rule and Metric Threshold Rule {kibana-pull}146874[#146874]
+* Adds new context variable called groupByKeys {kibana-pull}146633[#146633]
+* Adds new context variable for group by keys {kibana-pull}145654[#145654]
+* Adds Platinum license check for SLO APIs and SLO pages {kibana-pull}149055[#149055]
+* Create SLO / Edit SLO Form - Custom KQL {kibana-pull}147843[#147843]
+* SLO List {kibana-pull}147447[#147447]
+
+Platform::
+New trigger actions for chart legends and table cell actions {kibana-pull}146779[#146779]
+
+Querying & Filtering::
+* Insight filter builder form as markdown plugin {kibana-pull}150363[#150363]
+* Adds the ability to support complex filters with AND/OR relationships {kibana-pull}143928[#143928]
+
+Security::
+* Adds the ability to allow administrators to limit the number of concurrent user sessions with `xpack.security.session.сoncurrentSessions.maxSessions` {kibana-pull}147442[#147442]
+* API Keys can now be updated with new role descriptors and metadata in the API Keys Management screen {kibana-pull}146237[#146237]
+
+For more information about the features introduced in 8.7.0, refer to <<whats-new,What's new in 8.7>>.
+
+[[enhancements-and-bug-fixes-v8.7.0]]
+=== Enhancements and bug fixes
+For detailed information about the 8.7.0 release, review the enhancements and bug fixes.
+
+[float]
+[[enhancement-v8.7.0]]
+=== Enhancements
+Alerting::
+* Bring flapping status and settings in o11y {kibana-pull}150483[#150483]
+* Redesign all cases list select modal {kibana-pull}149851[#149851]
+* RenderCustomActionsRow with named params instead of args {kibana-pull}149304[#149304]
+* Adds new column `Updated on ` in `all cases list ` table. This column can be sorted and can persist sorting options {kibana-pull}149116[#149116]
```

Comment:
Same as before, I believe all cases items should be in a separate category called `Cases`.